build(cwf) fix vulnerability gorilla websocket version

nikolakn opened the pull request:
Signed-off-by: Nikola Knezevic <nkcodeplus@gmail.com>

fix issue:
https://github.com/magma/magma/security/dependabot/cwf/k8s/cwf_operator/go.sum/github.com%2Fgorilla%2Fwebsocket/open

## Summary

update dependency:
github.com/gorilla/mux v1.7.4 => github.com/gorilla/mux v1.8.0

## Test Plan

nikolakn committed:
build(cwf) fix vulnerability gorilla websocket version

Signed-off-by: Nikola Knezevic <nkcodeplus@gmail.com>

nikolakn committed:
build(cwf) fix vulnerability gorilla websocket version

Signed-off-by: Nikola Knezevic <nkcodeplus@gmail.com>

ssanadhya committed:
chore(mme): Remove unused function from SPGW context manager (#10184)

Signed-off-by: Shruti Sanadhya <ssanadhya@fb.com>

reginawang3495 committed:
feat: add possibility of singleton for singleton reindexer for prod (#10259)

pshelar committed:
refactor(pipelined): Remove egres flows from ARP controller. (#10261)

This reduces numbers of flows as well as code in ARP controller..

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

alexzurbonsen committed:
chore(directoryd): enable selective opt-in logging to Sentry for directoryd (#10113)

Adds the decorator to catch unexpected errors for directoryd's gRPC endpoints.

Signed-off-by: Alexander zur Bonsen <alexander.zur.bonsen@tngtech.com>

sebathomas committed:
chore(ctraced): Log unexpected errors in server in opt-in mode (#10110)

Adds the decorator to catch unexpected errors to gRPC endpoints.

Signed-off-by: Sebastian Thomas <sebastian.thomas@tngtech.com>
Signed-off-by: Alexander zur Bonsen <alexander.zur.bonsen@tngtech.com>

uri200 committed:
fix(sessiond): fix race condition on handle_activate_ue_flows_callback (#10211)

ardzoht committed:
fix(agw): Updating import path for load test protos (#10264)

Signed-off-by: Alex Rodriguez <alexrod@fb.com>

spikey979 committed:
fix(feg): Incorrect conversion between integer types (#10278)

* Added check if variable is within the allowable bounds

Signed-off-by: Kristijan <spikey979@gmail.com>

* Added check if variable is in bounds of Unsigned32 type (0-4294967295)

Signed-off-by: Kristijan <spikey979@gmail.com>

spikey979 committed:
fix(feg): Incorrect conversion between integer types (#10274)

* gy_client.go, utils.go and main.go solved incorrect conversion between integer types

Signed-off-by: Kristijan <spikey979@gmail.com>

* String was parsed into an int using strconv.Atoi which can cause incorrect conversion  if int is converted into another integer type of a smaller sizeA check was added to see if number is in limits for type byte (0-255)

Signed-off-by: Kristijan <spikey979@gmail.com>

themarwhal commented:
This might be relevant: https://github.com/magma/magma/pull/9458#issuecomment-968614941

We've seen a few cases of GHA jobs not triggering for certain users. Apparently, the only way to remediate is to contact GH.

themarwhal commented:
A known workaround, is to have someone who is not having this issue to just push to your wokring branch. That should trigger the jobs.

Neudrino committed:
ci(agw): Python3 is required in Bazel builds (#10245)

* ci(agw): python3 is needed for the for magma to run

For [focal fossa](https://github.com/magma/magma/blob/master/.devcontainer/Dockerfile#L2) [Python 3.8 is a requirement](https://github.com/magma/magma/blob/master/orc8r/gateway/python/python.mk#L24).

Signed-off-by: Fritz Lehnert <Fritz.Lehnert@tngtech.com>

* ci(agw): python3 is needed for the for magma to run

For [focal fossa](https://github.com/magma/magma/blob/master/experimental/bazel-base/Dockerfile#L4) [Python 3.8 is a requirement](https://github.com/magma/magma/blob/master/orc8r/gateway/python/python.mk#L24).
`python-dev` is deprecated according to `apt show python-dev-is-python3`.

Signed-off-by: Fritz Lehnert <Fritz.Lehnert@tngtech.com>

* refactor(agw): sort packages alphabetically ascending

Signed-off-by: Fritz Lehnert <Fritz.Lehnert@tngtech.com>

alexzurbonsen committed:
chore(state): enable selective opt-in logging to Sentry for state service (#10125)

In opt-in selected errors from GarbageCollector and StateReplicator are sent to Sentry.

Signed-off-by: Alexander zur Bonsen <alexander.zur.bonsen@tngtech.com>

themarwhal committed:
chore(ci): Always push test results and logs even with failure (#10315)

Signed-off-by: Marie Bremner <marwhal@fb.com>

themarwhal committed:
chore: remove code review for generated files (#10321)

Signed-off-by: Marie Bremner <marwhal@fb.com>

ulaskozat committed:
test(mme): network initiated detach timer expirations (#10290)

Signed-off-by: Ulas Kozat <kozat@fb.com>

ardzoht committed:
fix(agw): Fixing service request metrics count on Grafana (#10095)

* fix(agw): Fixing service request metrics count on Grafana

Signed-off-by: Alex Rodriguez <alexrod@fb.com>

* fix(agw): Fixing service request metrics count on Grafana

Signed-off-by: Alex Rodriguez <alexrod@fb.com>

themarwhal committed:
chore(mme): clean up oai-mme cmake files (#10324)

Signed-off-by: Marie Bremner <marwhal@fb.com>

reginawang3495 committed:
refactor: rename storeImpl to store (#10296)

talkhasib committed:
Clean AGW artifacts pushed to firebase (#10288)

Signed-off-by: Tariq Al-Khasib <talkhasib@fb.com>

ulaskozat committed:
chore(mme): Remove unused dump functions (#10335)

Signed-off-by: Ulas Kozat <kozat@fb.com>

uri200 committed:
chore(mme): remove possible redundancy at grpc_service cmake (#10292)

Signed-off-by: Oriol Batalla <obatalla@fb.com>

nikolakn committed:
Merge branch 'pr-fix-vulnerability-gorilla-websocket-version' of github.com:nikolakn/magma into pr-fix-vulnerability-gorilla-websocket-version
